### PR TITLE
Connection/Chat

### PR DESCRIPTION
**Pull request by NicKoehler**
As discussed in #4336 

Here's the changes:

- [x] Larger Chat
- [x] Grid layout
- [x] Fluent icons
- [x] Chat moved to left
- [x] Removed name from received messages
- [x] Other minor adjustments (Colors, Padding, Margins)

| Light | Dark |
| -- | -- |
| immagine | immagine |

### CODE REVIEW

**Comment by rustdesk** [comment]
👍

File: flutter/lib/common.dart [hunk 1]
```diff
@@ -196,6 +183,8 @@ class MyTheme {
     ),
   );
 
+  static const SwitchThemeData switchTheme = SwitchThemeData(splashRadius: 0);
```
Comment (grummbeer) [inline flutter/lib/common.dart:186]: No switch used. SwitchTheme can be removed?

Comment (grummbeer) [reply flutter/lib/common.dart:186]: Switches are used at android server/settings page, they might affected by this change. not sure.

**Comment by rustdesk** [comment]
Can we add off/on status to tooltip?

**Comment by grummbeer** [comment]
> how do you think about this?

The labeled switch/checkbox is the solution to exactly this problem. 

Even with e.g. red/green colored, it is a riddle whether green means "I am safe" or if permission was granted (possibly "I am not safe"). Same with grey/blue

**Comment by rustdesk** [comment]
I still think it can be solved visually. For example, add some mark on the icon as below. @michalwitek1232 do you have any idea for such design?

image

**Comment by grummbeer** [comment]
> I still think it can be solved visually. For example, add some mark on the icon as below 

:-) yeah good idea!

**Comment by NicKoehler** [comment]
Do you like something like this?

immagine

**Comment by rustdesk** [comment]
How about?

image

**Comment by NicKoehler** [comment]
What do you think?

immagine

**Comment by rustdesk** [comment]
> What do you think?

How about if removing ON?

**Comment by rustdesk** [comment]
> How about if removing ON?

Ignorethis, both "ON/OFF" looks better.

**Comment by michalwitek1232** [comment]
> I still think it can be solved visually. For example, add some mark on the icon as below (stack layout can be used to add these mark). @michalwitek1232 do you have any idea for such design?
> 
> image image



> What do you think?
> 
> immagine

I'm thinking about something like this maybe?

image

What this design tells user ?
- This could be enabled but right now is disabled

I mean, the problem with buttons as checkboxes/switches is that it's kind hard to signalize user that this could be flipped ON/OFF

**Comment by rustdesk** [comment]
Thanks @michalwitek1232

**Comment by rustdesk** [comment]
Since fluent icon pub increase the final exe file size a lot, I will remove fluent icon pub and use svg files.

**Comment by michalwitek1232** [comment]
> Since fluent icon pub increases the final exe file size a lot (1.3 M), I will remove fluent icon pub and use svg files.
> 
> image

That makes a lot of sense. Good move 👍

**Pull request merged**